### PR TITLE
lesson-10

### PR DESCRIPTION
**Pull request by DariaRomanowska**

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -362,6 +364,87 @@ input[type="submit"] {
...
+  color: #383838;
+}
+
+.gallery {
+  display: grid;
+  position: relative;
+  row-gap: 24px;
+  column-gap: 16px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(4, auto);
```
Comment (5tronciy) [inline styles/styles.css]: Implicit adding of margin-bottom value. It is not clear to understand.

File: styles/styles.css [hunk 2]
```diff
@@ -410,11 +493,13 @@ footer {
...
   /* For mobile phones: */
-  :root{
+  :root {
     --col-width: calc(100% / 6);
   }
 
-  *:after {display:none}
+  *:after {
+    display: none;
+  }
```
Comment (5tronciy) [inline styles/styles.css]: A rather extreme solution. People don't expect disabling pseudo-elements by default.

File: index.html [hunk 3]
```diff
@@ -304,12 +313,42 @@ <h2 class="title">What do we offer</h2>
         </div>
       </div>
     </div>
+    <div class="background-homes">
+      <div class="container homes">
+        <h2 class="homes-title">Homes guests loves</h2>
+        <div class="gallery">
```
Comment (5tronciy) [inline index.html:319]: I think a consequence of simplifying this block is the subsequent technique of hiding elements by `div.gallery p:nth-last-of-type(1...4)`. It is bad that the styles depend on the content there.

File: styles/styles.css [hunk 4]
```diff
@@ -487,11 +572,89 @@ footer {
...
+  .gallery p{
+    font-size: 12px;
+    line-height: 14px;
+  }
+
+  .gallery p:first-of-type{
+    text-shadow: none;
+  }
+
+  div.gallery img:nth-last-of-type(1){
```
Comment (5tronciy) [inline styles/styles.css]: You used class `gallery` before. Why clarify that it is div here?

**Review by 5tronciy — CHANGES_REQUESTED** [review]
The block "Homes guests loves" looks not bad. But the issue with the first section brokes the layout on low-width screens. Look throw my comments also.

File: styles/styles.css (hunk 2 above)
Comment (DariaRomanowska) [reply styles/styles.css]: They are not in the design, but in the mobile version, the hover will still not work. What can I do in this case?

File: styles/styles.css (hunk 1 above)
Comment (DariaRomanowska) [reply styles/styles.css]: I don't understand what do you mean

File: index.html (hunk 3 above)
Comment (DariaRomanowska) [reply index.html:319]: In styles, I used this method. What does styles depend on content mean? Aren't styles writtening to the content?

File: styles/styles.css (hunk 1 above)
Comment (5tronciy) [reply styles/styles.css]: There are 3 rows with content. And as you add the 4th row without content in the template there is an additional gap at the bottom. And it works as margin-bottom but you have written it in a non-obvious way.
Снимок экрана 2022-08-23 в 21 48 49

File: styles/styles.css (hunk 2 above)
Comment (5tronciy) [reply styles/styles.css]: What about using concrete classes instead of `*` (every element in the document)?

File: index.html (hunk 3 above)
Comment (5tronciy) [reply index.html:319]: Content is changeable like the design. In this case, your style works for a max of 4 tiles. 
And you wrote 6 selectors just for hiding 2 tiles. What if we decide to increase the number of tiles?
It is better to find ways of writing common styles independent of content.

**Review by 5tronciy — APPROVED** [review]
Looks good.
10th lesson's requirements are fully assigned.

**Pull request merged**